Keyboards/KeyboardsBase/ScribeFunctionality/Conjugate.swift

```diff
@@ -236,7 +239,7 @@ func returnConjugation(keyPressed: UIButton, requestedForm: String) {
 /// Returns the conjugation state to its initial conjugation based on the keyboard language.
 func resetVerbConjugationState() {
   conjViewShiftButtonsState = .leftInactive
-  if controllerLanguage == "French" {
+  if ["French_AZERTY", "French_QWERTY"].contains(controllerLanguage) {
```

Comment:
Makes sense :) I think the easiest way of doing this right now that would allow for variants to be added without needing to change them is to just pattern match on the string in the following way:

```swift
controllerLanguage.prefix("French".count) == "French"
```

Reply:
Love it :+1: Yep, I agree! I think that'd be a pretty clean way of doing it if the variants all begin with the target language descriptor

Reply:
Will have to, but I think it makes sense to do it that way anyway 😊